### DISCUSSION TITLE
How to remove a GitHub Page having removed the repository

### DISCUSSION THREAD

**Discussion by RaulCatalinas** [community/community#32968] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello everyone.
How can I remove a GitHub Page having removed the associated repository?
I have no idea how to do it.
Thanks in advance.

**Comment by radar-blip** [comment, marked as answer]
If you deleted the repository, the page should be gone. 
Have you checked to see if the page is still there, lately?

**Comment by RaulCatalinas** [comment]
It has already disappeared and it gives me a 404 error, thank you very much
